src/Resources/views/CRUD/edit_orm_many_association_script.html.twig

```diff
@@ -473,6 +473,11 @@ This code manage the many-to-[one|many] association field popup
                     jQuery('#field_widget_{{ id }}').html(html);
                 }
             });
+
+            {% if btn_edit %}
+                var edit_button_url = '{{ sonata_admin.field_description.associationadmin.generateUrl('edit', {'id' : 'OBJECT_ID'}) }}'.replace('OBJECT_ID', jQuery(this).val());
```

Comment:
Hard to review from phone, please linebreak

Reply:
@greg0ire my bad, fixed now.